### PR TITLE
Add workdir cmdline arg

### LINKED ISSUE #156: user should be able to set the working dir with `--workdir`
### Description

A user should have the ability to override all empty tests `dir` with a command line argument such as `--workdir`.

This could require a major version bump. Currently in `config` section of a suite file workdir is known as `dir`. It would be nice if these matched.

We may want to  complete #131 as well before we do the version bump.

### PR DESCRIPTION
**Pull request by dylanhitt**
Fixes #156 

## Checklist

 - [x] Added unit / integration tests for windows, macOS and Linux? 
 - [x] Added a changelog entry in [CHANGELOG.md](https://github.com/commander-cli/commander/blob/master/CHANGELOG.md)?
 - [x] Updated the documentation ([README.md](https://github.com/commander-cli/commander/blob/master/README.md), [docs](https://github.com/commander-cli/commander/blob/master/docs))?
 - [x] Does your change work on `Linux`, `Windows` and `macOS`?

### CODE REVIEW

**Comment by dylanhitt** [comment]
@SimonBaeumer I'll probably have the prototype PR up for the `Test_Command` refactor by this weekend. 

Let me know if this needs any changes. Best

File: pkg/app/test_command_test.go [hunk 1]
```diff
@@ -178,6 +178,11 @@ func Test_ConvergeResults(t *testing.T) {
 	assert.Equal(t, 1, actual.Failed)
 }
 
+func Test_TestCommand_InvalidDir(t *testing.T) {
+	err := TestCommand("foo", TestCommandContext{Workdir: "invalid_dir"})
```
Comment (SimonBaeumer) [inline pkg/app/test_command_test.go:182]: this needs a check for windows like 
```
if runtime.GOOS == "windows" {...}
````

**Comment by SimonBaeumer** [comment]
LGTM, only one small change necessary in the unit tests for windows

**Review by SimonBaeumer — APPROVED** [review]
Nice!
@dylanhitt feel free to merge it

**Pull request merged**